docs/installation/prepare/microcontrollers/_index.md

```diff
@@ -1,387 +1,298 @@
+cargo install ldproxy
+```
+
+Finally to download and install the esp-rs toolchain run :
+
+```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
+espup install -s -f ~/esp/export-rs.sh -v 1.67.0
+```
+
+To activate the ESP Rust toolchain, run the following command to source (`.`) the activation script `export-rs.sh`:
+
+```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
+. $HOME/esp/esp-idf/export-rs.sh
+```
+
+To to avoid conflicts with other toolchains, adding this command to your `.bashrc` or `.zshrc` is not recommended.
+Save this command to run in any future terminal session where you need to activate the ESP Rust development framework.
+
+## Install the Micro-RDK
+
+To install the Micro-RDK on your ESP32 board:
+
+- [create a new robot in the Viam app](#create-a-new-robot)
+- [generate a new project from Viam's micro-RDK template](#generate-a-new-project-from-the-micro-rdk-template)
+- [upload the project to your ESP32](#upload-and-connect-to-your-esp32-board)
+
+### Create a New Robot
+
+Navigate to [the Viam app](https://app.viam.com) and create a new robot in your desired location.
+Keep your `Mode` and `Architecture` selections at default, and skip the instructions in the **Setup** tab for now as the setup instructions there are not for microcontrollers.
+You can configure your robot using the config tab.
+
+{{< alert title="Caution" color="caution" >}}
+As of the time this guide is written the esp32 board component doesn't exist yet on app.viam.com below you can find the board template for esp32:
+
+```json
+{
+  "attributes": {
+    "webhook": "", // Optional path a webhook see https://github.com/viam-labs/webhook-template for an example
+    "webhook-secret": "", // Option webhook secret
+    "pins": [ // pin to be configured as Input/Ouput (for Get/SetGpio)
+      15
+    ],
+    "analogs": [
+      {
+        "pin": "34", // analog pins
+        "name": "sensor"
+      }
+    ]
+  },
+  "depends_on": [],
+  "model": "esp32",
+  "name": "board",
+  "type": "board"
+}
+```
+
+{{% /alert %}}
+
+### Generate a New Project from the Micro-RDK Template
+
+Use [the micro-RDK template](https://github.com/viamrobotics/micro-rdk-template.git) to create a new micro-RDK project to upload to your ESP32 by running:
+
+```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
+cargo generate --git https://github.com/viamrobotics/micro-rdk-template.git
+```
+
+If you would like, you can use `mkdir` to initialize a new repository in the directory you created by running `cargo-generate`, to track any changes you make to the generated project.
+
+You will be prompted to paste your robot's JSON configuration into the terminal.
+
+To obtain this, navigate to [the Viam app](https://app.viam.com).
+Click the **Copy viam-server config** button on the right side of the **Setup** tab of your robot.
+The Micro-RDK uses the config for communication with the Viam app.
```

Comment:
Sorry as this might just be my misunderstanding- what does this sentence mean? config is sent to the app bc the robot needs a config file?

Reply:
This files contains the secret needed for the esp32 to connect to app.viam.com